base/nullable.jl

```diff
@@ -3,7 +3,7 @@
 immutable NullException <: Exception
 end
 
-Nullable{T}(value::T, isnull::Bool=false) = Nullable{T}(value, isnull)
+Nullable{T}(value::T, hasvalue::Bool=true) = Nullable{T}(value, hasvalue)
```

Comment:
We make use of this for null-safe operations where we can generate the `value` and `hasvalue` fields without branching.